optaplanner-examples/src/main/java/org/optaplanner/examples/common/persistence/generator/LocationDataGenerator.java

```diff
@@ -68,53 +73,10 @@
             new LocationData("STR", 48.69, 9.221944),
             new LocationData("OPO", 41.235556, -8.678056),
             new LocationData("KBP", 50.344722, 30.893333),
-            new LocationData("VCE", 45.505278, 12.351944),
-    };
-
-    public static final LocationData[] EUROPE_CAPITALS = {
```

Comment:
This appears to have been unused.

Reply:
Please don't throw it away, it was used for the europe40 dataset and it was a PITA to collect that data.